pokemon/pokeParser.py

```diff
@@ -0,0 +1,14 @@
+import re
+
+pokedex = open("pokemon\\pokemon.html")
+
+regex = re.compile("href=\"(/wiki/\w+)\".+title=\"(\w+)\"")
+
+pokedict = {}
+i = 0
+
+for match in re.finditer(regex, pokedex.read()):
```

Comment:
```suggestion
for index, match in enumerate(re.finditer(regex, pokedex.read())):
```
you can remove your counter.